plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergV2.java

```diff
@@ -798,13 +995,20 @@ private void writeEqualityDeleteToNationTable(Table icebergTable, Optional<Parti
 
     private void writeEqualityDeleteToNationTable(Table icebergTable, Optional<PartitionSpec> partitionSpec, Optional<PartitionData> partitionData, Map<String, Object> overwriteValues)
             throws Exception
+    {
+        writeEqualityDeleteToNationTableWithDeleteColumns(icebergTable, partitionSpec, partitionData, overwriteValues, Optional.empty());
+    }
+
+    private void writeEqualityDeleteToNationTableWithDeleteColumns(Table icebergTable, Optional<PartitionSpec> partitionSpec, Optional<PartitionData> partitionData, Map<String, Object> overwriteValues, Optional<List<String>> deleteFileColumns)
```

Comment:
@jasonf20 @findinpath check this. I add a param deleteFIleColumns, we can use to equality delete file with nested column.